vendor/brave-ios/Ledger/BATBraveLedger.mm

```diff
@@ -36,10 +36,13 @@
 #import "base/command_line.h"
 #import "base/i18n/icu_util.h"
 #import "base/ios/ios_util.h"
+
+#include "base/callback_helpers.h"
 #include "base/sequenced_task_runner.h"
 #include "base/task/post_task.h"
 #include "base/task/thread_pool.h"
 #include "base/task_runner_util.h"
+#include "mojo/public/cpp/bindings/remote.h"
```

Comment:
brave-ios BUILD.gn will need to add `//mojo/public/cpp/bindings` as a dep, no?

Reply:
Yep, thanks.